### DISCUSSION TITLE
How do I stop changesets trying to publish certain packages

### DISCUSSION THREAD

**Discussion by dagda1** [changesets/changesets#783] [Q&A]
I misunderstood the `"ingore"` arrray in changes.json.

I have mine like this:

```json
  "ignore": [
    "@cutting/graphql-explorer",
    "@cutting/website"
  ]
  ```

I thought this would stop changesets from trying to publish these packages but it just stops the versions being bumped.

Is there anyway to stop packages getting published?

**Comment by Andarist** [comment]
You can set `private: true` in your `package.json`. Note that I think that we probably should also respect `ignore` when publishing and I would accept a PR implementing this 😉

**Reply by kachkaev** [reply]
👋 @Andarist! I started a PR: https://github.com/changesets/changesets/pull/866
